src/RestServer/RestWebServer.cs

```diff
@@ -0,0 +1,298 @@
+                            options.MimeTypes = ResponseCompressionDefaults.MimeTypes.Append(MediaTypeNames.Application.Json);
+                        });
+
+                    var controllers = services
+                        .AddControllers(options =>
+                        {
+                            options.EnableEndpointRouting = false;
+                        })
+                        .ConfigureApiBehaviorOptions(options =>
+                        {
+                            options.InvalidModelStateResponseFactory = context =>
+                                new BadRequestObjectResult(
+                                    new ParameterFormatExceptionModel(string.Join(' ', context.ModelState.Values.SelectMany(s => s.Errors).Select(s => s.ErrorMessage))))
+                                {
+                                    ContentTypes =
+                                        {
+                                            MediaTypeNames.Application.Json,
+                                        }
+                                };
+                        })
+                        .ConfigureApplicationPartManager(manager =>
+                        {
+                            var controllerFeatureProvider = manager.FeatureProviders.Single(p => p.GetType() == typeof(ControllerFeatureProvider));
+                            var index = manager.FeatureProviders.IndexOf(controllerFeatureProvider);
+                            manager.FeatureProviders[index] = new BlackListControllerFeatureProvider(_settings);
+
+                            foreach (var plugin in Plugin.Plugins)
+                                manager.ApplicationParts.Add(new AssemblyPart(plugin.GetType().Assembly));
+                        })
+                        .AddNewtonsoftJson(options =>
+                        {
+                            options.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
+                            options.SerializerSettings.Formatting = Formatting.None;
+
+                            foreach (var converter in _settings.JsonSerializerSettings.Converters)
+                                options.SerializerSettings.Converters.Add(converter);
+                        });
+
+                    if (_settings.EnableSwagger)
+                    {
+                        services.AddSwaggerGen(options =>
+                        {
+                            options.SwaggerDoc("v1", new OpenApiInfo()
+                            {
+                                Title = "RestServer API - V1",
+                                Description = "Neo-cli's Rest API.",
+                                Version = "v1",
+                                License = new OpenApiLicense()
+                                {
+                                    Name = "MIT",
+                                    Url = new Uri("http://www.opensource.org/licenses/mit-license.php"),
```

Comment:
What license should we be using for this? I currently have MIT license. This is for Swaggers, _example below_
![image](https://github.com/neo-project/neo-modules/assets/8141309/1bcffdd8-8734-492d-8196-4b9861c782fe)